example/centos/pipeline/image/s390x.yaml

```diff
@@ -0,0 +1,59 @@
+name: image
+build: name:build
+stages:
+  otk.op.join:
+    values:
+      - otk.external.otk-make-partition-stages:
+          ${filesystem}
+      - - type: org.osbuild.copy
+          inputs:
+            root-tree:
+              type: org.osbuild.tree
+              origin: org.osbuild.pipeline
+              references:
+                - name:os
+          options:
+            paths:
+              - from: input://root-tree/
+                to: mount://-/
+          devices:
+            otk.op.join:
+              values:
+                - ${fs_options.devices}
+                # TBD this seems strange:
+                - disk:
+                    type: 'org.osbuild.loopback'
+                    options:
+                      filename: 'disk.img'
+          mounts:
+            ${fs_options.mounts}
+        - type: org.osbuild.zipl.inst
+          options:
+            kernel: 8-2.fk1.s390x
+            location: 2048
+          devices:
```

Comment:
Please have a look at the other image pipeline examples, it should be possible to use something like
```yaml
          devices:
            ${fs_options.devices}
          mounts:
            ${fs_options.mounts}
```
here.

With all the hardcoded sizes in the fragement this will fail as soon as any of the size changes so these need to be dynamically caludlated via an external (and there is support for this already).

Reply:
As commented in the PR in "images" I wasn't aware that the `fs_options.*` structure is fully compatible with `org.osbuild.zipl`. Thanks for pointing this out!

Reply:
code updated, please check